### ISSUE TITLE
Can't reproduce the DCASE2020 EfficientNet-B0 baseline (0.583, Table VI): I get 0.457 / 0.519

### ISSUE THREAD

**Issue by OnnoNiemann** [ramppdev/sample-difficulty-curriculum-learning#1]
Hi Simon, I talked to Manuel at IJCNN and watched his presentation and now I am trying to work with your code. 

First of all, very nice and clear documentation :)

For some reason I don't get the random-init EfficientNet-B0 baseline on DCASE2020 performance of 58.3 from Table VI (SAM, lr 0.01, batch size 16, 50 epochs, seed 1). I followed the README and get lower numbers.

What I ran

cd dcase
aucurriculum fetch -cn train
aucurriculum preprocess -cn train
aucurriculum train -cn train_baselines   # restricted to +seed: 1, model EfficientNet-B0, SAM-SGD-M9, lr 0.01

Results (seed 1)

┌────────────────────────────────────────────────────────────────────────────────────┬─────────────────────────────────────┐
│                                       Setup                                        │ test_accuracy (best-dev checkpoint) │
├────────────────────────────────────────────────────────────────────────────────────┼─────────────────────────────────────┤
│ Repo as-is (autrainer default DCASE2020Task1A-32k.yaml, dev_split: 0.1)            │ 0.457                               │
├────────────────────────────────────────────────────────────────────────────────────┼─────────────────────────────────────┤
│ Local dcase/conf/dataset/DCASE2020Task1A-32k.yaml with dev_split: 0.0 (dev = test) │ 0.519                               │
└────────────────────────────────────────────────────────────────────────────────────┴─────────────────────────────────────┘

Is there something I'm doing wrong?

I changed the dev_split to 0.0 after I didn't get the performance with 0.1. In the paper you say a train-test split was used "without an explicit validation subset", for both datasets. For CIFAR-10 the repo sets this explicitly (cifar/conf/dataset/CIFAR-10.yaml, dev_split: 0.0). For DCASE there is no dataset override, so autrainer's default dev_split: 0.1 is used. Which setting produced the numbers in the paper?

Also in both runs it seems that the model overfits. Train loss ends at about 0.12–0.15 and dev loss at about 2.3–2.9. Dev accuracy stops improving at around 0.45–0.50 after the first few epochs.

Do you have ideas of what I might change? 
 
Thanks for your help! :)

**Comment by ramppdev** [comment]
Hi Otto,
thanks for reaching out!
Definitely sounds strange, and yes, you are correct, i forgot to override the `dev_split` to be `0.0` instead of `0.1`. I will correct that.

I will investigate the performance differences on my end, could you also provide me with the exact torch version and GPU setup you used?
